apps/moodytunes/templates/browse.html

```diff
@@ -19,6 +19,7 @@
 <h1>Browse Songs</h1>
 <div id="playlist-generator-container">
     {% include 'full_songs_info.html' %}
+    <p><a href="{% url 'moodytunes:suggest' %}">Suggest a new song for our site</a></p>
```

Comment:
Let's move this to a separate tab in the navbar, the browse interface is getting cluttered as is